### PR TITLE
Update user creation in README

### PR DESCRIPTION
**Pull request by mtekel**
## What

A follow-up to our discussion this morning - we do require real email addresses for users, so that there is someone to respond to our notifications/queries (in case we need to notify/reach user "owner" - if it's e.g. a CI user). Also update outdated email verification instructions.

## How to review

Proofread.

## Who can review
not @mtekel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -275,16 +277,40 @@ The script will create the organisation if it doesnt exist, create the user and
 email them the password. You must have the aws cli and cf cli installed, and be
 logged into the cf environment you want to create users for
 
+### Creating new accounts
+
+When receiving a request to create a new account, we do require that the email address
+is a real and reachabe address even for accounts that are meant for automated services
```
Comment (timmow) [inline README.md]: typo - reachable

Comment (mtekel) [inline README.md]: thanks, fixed

**Comment by mtekel** [comment]
Updated based on in-person comments from @urmishah

File: README.md [hunk 2]
```diff
@@ -275,16 +277,39 @@ The script will create the organisation if it doesnt exist, create the user and
...
 pass the `-m` option to the script.
 
 ### Verifying email address
 
-To run the above script, you need to verify your email address with [AWS SES](https://aws.amazon.com/ses/)
-To do this, run `aws ses verify-email-identity --email-address youremail@example.com --region eu-west-1`
-and click on the link in the resulting email
+User creation script sends notification to user with a temporary password upon creation or reset.
+Script uses our support email address (gov-uk-paas-support@digital.cabinet-office.gov.uk) as sender,
+so that any replies of users go to our support. Scrip uses [AWS SES](https://aws.amazon.com/ses/) to
```
Comment (timmow) [inline README.md]: Typo - scrip

**Pull request merged**